backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java

```diff
@@ -50,11 +49,6 @@ public class DeviceType extends EternalAuditedEntity {
   @Column(nullable = false)
   private int testLength;
 
-  @JsonIgnore
-  /** This relationship is necessary for DeviceTypeRepository.findAllByTestOrdersInternalIdIn */
-  @OneToMany(mappedBy = "deviceType", fetch = FetchType.LAZY)
```

Comment:
I find it odd that this worked without a corresponding column on the `deviceType` schema definition 🤔 Changes look good, I just wouldn't expect JPA to tolerate this.

Reply:
yeah honestly it didn't make a lot of sense to me too, anyway it would be very problematic, seems like a performance nightmare to pull every testorder that used a particular device

Reply:
Exactly! Very strange all around, so good find on the cleanup 👍